test/jdk/java/lang/Thread/virtual/stress/Skynet.java

```diff
@@ -21,19 +21,25 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2023, 2023 All Rights Reserved
+ * ===========================================================================
+ */
+
 /*
  * @test id=default
  * @summary Stress test virtual threads with a variation of the Skynet 1M benchmark
  * @requires vm.continuations
  * @requires !vm.debug | vm.gc != "Z"
- * @run main/othervm/timeout=300 -Xmx1g Skynet
+ * @run main/othervm/timeout=300 -Xmx1g -Xnocompressedrefs Skynet
  */
 
 /*
  * @test id=ZSinglegen
  * @requires vm.debug == true & vm.continuations
  * @requires vm.gc.ZSinglegen
- * @run main/othervm/timeout=300 -XX:+UnlockDiagnosticVMOptions
+ * @run main/othervm/timeout=300 -XX:+UnlockDiagnosticVMOptions -Xnocompressedrefs
```

Comment:
This and the next one don't need to be modified since they require vm.gc.ZSinglegen and vm.gc.ZGenerational

Reply:
Updated.